src/options/optionsMain.js

```diff
@@ -154,8 +154,8 @@ function optionsFactory(inpOptions, callback) {
 
 function sanitizeFilename(str) {
     const cleaned = sanitizeFilenameLib(str);
-    // remove non ascii
-    return cleaned.replace(/[^\x00-\x7F]/, '');
+    // remove all non ascii or do nothing
+    return cleaned.replace(/[^\x00-\x7F]/g, '') || cleaned;
```

Comment:
Hi, really sorry for the late reply. The reason why we don't want to keep non ascii characters in the filename is because it causes errors with building for windows while on macOS. What about using the `DEFAULT_APP_NAME` declared in line 19 of the same file as the default value instead?